src/config/partners/zora.ts

```diff
@@ -68,13 +65,13 @@ const zora: Partner = {
       image: '/partners/zora/drops/FINAL_ART-MalihaAbidi.jpg',
       creator: '0x7be470E6DC93855338a4Cc85D47447811421C7c8',
       name: 'a paragon',
```

Comment:
not sure this is intentional
```suggestion
      name: 'A Paragon',
```

Reply:
GTM tracker sheet has it as all lowercase 🤷